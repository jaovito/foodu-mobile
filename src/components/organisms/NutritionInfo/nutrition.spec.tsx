import {render} from '@test-utils';
import {NutritionInfo} from '.';

describe('<NutritionCard /> tests', () => {
  it('should be defined', () => {
    const nutritionInfo = render(<NutritionInfo />);

    expect(nutritionInfo).toBeDefined();
  });

  it('should be able to match with snapshot', () => {
    const nutritionInfo = render(<NutritionInfo />).toJSON();

    expect(nutritionInfo).toMatchSnapshot();
  });
});
