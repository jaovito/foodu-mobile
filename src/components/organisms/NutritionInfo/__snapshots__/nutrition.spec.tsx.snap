// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NutritionCard /> tests should be able to match with snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "width": "100%",
      },
    ]
  }
>
  <View
    style={
      [
        {
          "borderColor": "#FE8C00",
          "borderRadius": 20,
          "borderStyle": "solid",
          "borderWidth": 1,
          "minWidth": 177,
          "paddingBottom": 16,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 16,
        },
        {
          "marginTop": 63,
        },
      ]
    }
  >
    <Text
      fontWeight="medium"
      style={
        [
          {
            "color": "#0D0D0D",
            "fontFamily": "Poppins_500Medium",
            "fontSize": 31,
          },
          {
            "color": "#FE8C00",
            "fontSize": 27,
            "marginBottom": 16,
          },
        ]
      }
    >
      food.size
    </Text>
    <Text
      fontWeight="medium"
      style={
        [
          {
            "color": "#0D0D0D",
            "fontFamily": "Poppins_500Medium",
            "fontSize": 31,
          },
          {
            "color": "#0D0D0D",
            "fontSize": 35,
          },
        ]
      }
    />
  </View>
  <View
    style={
      [
        {
          "borderColor": "#FE8C00",
          "borderRadius": 20,
          "borderStyle": "solid",
          "borderWidth": 1,
          "minWidth": 177,
          "paddingBottom": 16,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 16,
        },
        {
          "marginTop": 63,
        },
      ]
    }
  >
    <Text
      fontWeight="medium"
      style={
        [
          {
            "color": "#0D0D0D",
            "fontFamily": "Poppins_500Medium",
            "fontSize": 31,
          },
          {
            "color": "#FE8C00",
            "fontSize": 27,
            "marginBottom": 16,
          },
        ]
      }
    >
      food.energy
    </Text>
    <Text
      fontWeight="medium"
      style={
        [
          {
            "color": "#0D0D0D",
            "fontFamily": "Poppins_500Medium",
            "fontSize": 31,
          },
          {
            "color": "#0D0D0D",
            "fontSize": 35,
          },
        ]
      }
    />
  </View>
  <View
    style={
      [
        {
          "borderColor": "#FE8C00",
          "borderRadius": 20,
          "borderStyle": "solid",
          "borderWidth": 1,
          "minWidth": 177,
          "paddingBottom": 16,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 16,
        },
        {
          "marginTop": 63,
        },
      ]
    }
  >
    <Text
      fontWeight="medium"
      style={
        [
          {
            "color": "#0D0D0D",
            "fontFamily": "Poppins_500Medium",
            "fontSize": 31,
          },
          {
            "color": "#FE8C00",
            "fontSize": 27,
            "marginBottom": 16,
          },
        ]
      }
    >
      food.delivery
    </Text>
    <Text
      fontWeight="medium"
      style={
        [
          {
            "color": "#0D0D0D",
            "fontFamily": "Poppins_500Medium",
            "fontSize": 31,
          },
          {
            "color": "#0D0D0D",
            "fontSize": 35,
          },
        ]
      }
    />
  </View>
</View>
`;
